tests/data/valid-statements/statement_annotations_transformedContent.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "statementId": "2f7bf9370f1254068e5e946df067d07d",
+        "declarationSubject": "xyz",
+        "statementDate": "2017-11-18",
+        "annotations": [
+            {
+                "statementPointerTarget": "/recordDetails/interestedParty",
```

Comment:
not sure this makes sense with this being an entity statement, it doesn't have this field?